src/components/result/FilterSelect/index.tsx

```diff
@@ -1,58 +1,56 @@
-import { useMedia } from '@offer-ui/react'
-import { useEffect, useState } from 'react'
 import type { ReactElement } from 'react'
 import { Styled } from './styled'
 import type { FilterSelectProps } from './types'
 import { PriceDialog } from '../PriceDialog'
-
+import { useGetSortOptions } from '@apis/post/queries'
 const FilterSelect = ({
   categoryItems,
-  sortPriceItems,
   tradePeriodItems,
   selectedCategoryValue,
+  selectedTradePeriodValue,
   minPriceValue,
   maxPriceValue,
+  applyPrice,
+  postData,
   handleSortPriceChange,
   handleCategoryChange,
   handleTradePeriodChange,
   handleMinPriceInputChange,
   handleMaxPriceInputChange,
   handlePriceApplyClick
 }: FilterSelectProps): ReactElement => {
-  const { tablet, mobile } = useMedia()
+  const { data: priceSortOptions, isLoading } = useGetSortOptions({
```

Comment:
```suggestion
  const sortOptions = useGetSortOptions({
```
페이지 내에 query와 mutation이 많아질 때, 네이밍을 변경해서 사용하는 형태가 복잡해 보여서 네이밍 변경 없이 객체로 받아서 사용하는 것이 좋을 것 같습니다!

Reply:
useGetSortOptions 제거로 인해 해당 리뷰 링크 스킵하겠습니다!